### ISSUE TITLE
Logged-in window goes beyond desktop scope if with moderator tabs

### ISSUE THREAD

**Issue by Bugtracker319** [Cockatrice/Cockatrice#4510]
**System Information:** 

Client Version: 2.8.1-custom(2fe572c) (2021-07-27)
Client Operating System: Ubuntu 20.04.3 LTS
Build Architecture: 64-bit
Qt Version: 5.12.8
System Locale: en_US
Install Mode: Standard
Highest screen-resolution: 1366 x 768
_______________________________________________________________________________________

**Description:**

Normal:
 - Not logged in -- Cockatrice client window size and fitting is correct, for both fullscreen and non-fullscreen; adjustable size (incl. height) of window in non-fullscreen.
 - Logged in (before, as non-mod) -- Same.

Issue:
 - Logged in (as mod; with the two new tabs etc.) -- Cockatrice client window goes beyond the scope of my desktop screen at the bottom, rendering the bottom ~10% inaccessible, including functions and any chat-input lines, for both full-screen and non-fullscreen; can't enter fullscreen mode of the window after un-fullscreen-ing when logged in (potentially independent of this issue); can't make the window shorter / reduce the window's height in non-fullscreen.

For additional reference see attached screenshots.
[CockatriceWindowBug_Screenshots.zip](https://github.com/Cockatrice/Cockatrice/files/7803223/CockatriceWindowBug_Screenshots.zip)
_______________________________________________________________________________________

**Steps to reproduce:**

 - Become a moderator (or similar administrative position) on a Cockatrice server, gaining additional tabs to the client interface in the process
 - Log in onto a server as moderator
 - Client window automatically escapes the screen scope and becomes inaccessible with or without using fullscreen mode; height of the window becomes unadjustable while logged in
 - Log out to be able to correct the window size when not in fullscreen; or it jumps automatically back to normal size when in fullscreen
_______________________________________________________________________________________

**Additional remarks:**

Issue potentially related to hardware limitations. Reports of similar cases however seem to exist (see Cockatrice-Discord, #support, 2022-01-02) but were mostly resolvable with repeated window-size adjustments.

**Comment by Bugtracker319** [comment]
Issue fixed for me with latest package (https://github.com/Cockatrice/Cockatrice/actions/runs/1671194358, specifically Focal).
Unless issue still persists with others and would come up again after testing I will close the ticket by Sunday, 2022-01-09, 22:00(UTC+0).

**Comment by tooomm** [comment]
>Issue fixed for me with latest package ([Cockatrice/Cockatrice/actions/runs/1671194358](https://github.com/Cockatrice/Cockatrice/actions/runs/1671194358), specifically Focal).

I think you linked a wrong CI run here? :)

>Unless issue still persists with others or would come up again after testing I will close the ticket by Sunday, 2022-01-09, 22:00(UTC+0).

Please do not close any tickets on your own as long as the issue is still persistent on the master branch.
Ebbit proposed a potential solution in a pull request so far. If it gets approved (and therefore your testing and feedback is highly appreciated!), the issue will get closed automatically once the fix is merged into master.

Then it's still waiting to be available for easy installation in the client via the build-in updater (beta channel first) with the next release.

**Comment by Bugtracker319** [comment]
> > Issue fixed for me with latest package ([Cockatrice/Cockatrice/actions/runs/1671194358](https://github.com/Cockatrice/Cockatrice/actions/runs/1671194358), specifically Focal).
> I think you linked a wrong CI run here? :)
You're right, I wanted to track it from the main branch, and then had a tab explosion. This is the correct run:
https://github.com/Cockatrice/Cockatrice/actions/runs/1670015276

> Please do not close any tickets on your own as long as the issue is still persistent on the master branch.
I apologise, I got overzealous.
